src/main/java/nextstep/session/domain/ImageReSolution.java

```diff
@@ -0,0 +1,34 @@
+package nextstep.session.domain;
+
+public class ImageReSolution {
+
+	private static final int WIDTH_RATIO = 3;
+	private static final int HEIGHT_RATIO = 2;
+	private int width;
```

Comment:
```suggestion
	private final int width;
```
인스턴스 변수는 상태 값이 변경되지 않는 경우 final로 구현할 것을 추천